app/src/main/res/layout/activity_event_details.xml

```diff
@@ -40,6 +39,13 @@
 
         </android.support.design.widget.AppBarLayout>
 
+        <FrameLayout
+            android:id="@+id/flContainer"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            app:layout_behavior="@string/appbar_scrolling_view_behavior"
+            tools:layout="@layout/fragment_profile_list"/>
```

Comment:
I thought we are going to have a facepile of images. Clicking on them take you to the attendees list fragment. I still think that's the best approch